### PR TITLE
Add GraphQL and SQL to License section in README

### PR DESCRIPTION
**Pull request by brfrn169**
Please take a look!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,4 +57,7 @@ Here are the contributors we are especially thankful for:
 - [Yonezawa-T2](https://github.com/Yonezawa-T2) - reported bugs around Serializable and proposed a new Serializable strategy (now named Extra-Read)
 
 ## License
-Scalar DB is dual-licensed under both the Apache 2.0 License (found in the LICENSE file in the root directory) and a commercial license. You may select, at your option, one of the above-listed licenses. The commercial license includes enterprise-grade tools, such as a multi-table consistent backup/restore tool for Cassandra. Regarding the commercial license, please [contact us](https://scalar-labs.com/contact_us/) for more information.
+Scalar DB is dual-licensed under both the Apache 2.0 License (found in the LICENSE file in the root directory) and a commercial license.
+You may select, at your option, one of the above-listed licenses.
+The commercial license includes enterprise-grade tools, such as a multi-table consistent backup/restore tool for Cassandra, and the GraphQL and SQL features.
```
Comment (brfrn169) [inline README.md:62]: Just added `, and the GraphQL and SQL features` in this line.

File: docs/index.md [hunk 2]
```diff
@@ -57,4 +57,7 @@ Here are the contributors we are especially thankful for:
 - [Yonezawa-T2](https://github.com/Yonezawa-T2) - reported bugs around Serializable and proposed a new Serializable strategy (now named Extra-Read)
 
 ## License
-Scalar DB is dual-licensed under both the Apache 2.0 License (found in the LICENSE file in the root directory) and a commercial license. You may select, at your option, one of the above-listed licenses. The commercial license includes enterprise-grade tools, such as a multi-table consistent backup/restore tool for Cassandra. Regarding the commercial license, please [contact us](https://scalar-labs.com/contact_us/) for more information.
+Scalar DB is dual-licensed under both the Apache 2.0 License (found in the LICENSE file in the root directory) and a commercial license.
+You may select, at your option, one of the above-listed licenses.
+The commercial license includes enterprise-grade tools, such as a multi-table consistent backup/restore tool for Cassandra, and the GraphQL and SQL features.
```
Comment (brfrn169) [inline docs/index.md:62]: Ditto.

**Review by Torch3333 — DISMISSED** [review]
LGTM, thank you!

File: README.md (hunk 1 above)
Comment (feeblefakie) [inline README.md:62]:
```suggestion
The commercial license includes several enterprise-grade features such as management tools and declarative query interfaces like GraphQL and SQL interfaces.
```

File: docs/index.md (hunk 2 above)
Comment (feeblefakie) [inline docs/index.md:62]:
```suggestion
The commercial license includes several enterprise-grade features such as management tools and declarative query interfaces like GraphQL and SQL interfaces.
```

**Review by feeblefakie — COMMENTED** [review]
Thank you!
Left some suggestions since the previous description is a bit stale. PTAL!

**Comment by brfrn169** [comment]
@feeblefakie Thank you! I applied your suggestions. Merging this.

**Pull request merged**